log-configs/connect-to-cluster.yaml

```diff
@@ -4,8 +4,19 @@
 rotation:
     logLimit: 104857600 # 100MB
     keepFiles: 100
+
 loggerTree:
-  severity: Debug+
-  files:
-    - node.pub
-    - node
+  severity: Info+
+
+  handlers:
+    - { name: "Public"
+      , filepath: "pub/node.log"
+      , logsafety: PublicLogLevel
+      , severity: Debug+
+      , backend: FileTextBE }
+    - { name: "Secret"
+      , filepath: "node"
+      , logsafety: SecretLogLevel
+      , severity: Info+
+      , backend: FileTextBE }
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/log-configs/connect-to-cluster.yaml#L8-L22